shared/chat/conversation/rekey/enter-paper-key.js

```diff
@@ -17,7 +16,6 @@ export default compose(
       onBack: () => dispatch(navigateUp()),
       onEnterPaperkey: (key: string) => {
         dispatch(checkPaperKey(new HiddenString(key)))
-        dispatch(loadInbox(true))
```

Comment:
we don't need to load the inbox again, the daemon tells us when things have been updated automatically